docs/node/getting-started.md

```diff
@@ -39,60 +39,83 @@ $ npm install @bufbuild/buf @bufbuild/protoc-gen-es @bufbuild/protobuf @connectr
 
 ## Define a service
 
-First, we need to add a Protobuf file that includes our service definition. For
-this tutorial, we are going to construct a unary endpoint for a service that is
-a stripped-down implementation of [ELIZA](https://en.wikipedia.org/wiki/ELIZA),
-the famous natural language processing program.
+We have already defined the `ElizaService` and have
+it [hosted on the Buf Schema Registry][eliza-proto] to use in this example.
+Export the Protobuf schema to use in our project.
```

Comment:
With Connect for Node.js, users can serve RPCs. We're showing how to do that in this tutorial. It's most common that users who implement a service also author the schema. In this context, I'd prefer that we let the user author greet.proto, instead of exporting it from the BSR. Similar to what we do in the connect-go docs.